src/components/Contexts/OrganizationListContext.test.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { FC } from "react";
+import { render, waitFor } from "@testing-library/react";
+import { MockedProvider, MockedResponse } from "@apollo/client/testing";
+import { GraphQLError } from "graphql";
+import {
+  OrganizationProvider,
+  Status as OrgStatus,
+  useOrganizationListContext,
+} from "./OrganizationListContext";
+import { LIST_ORGS } from "../../graphql";
+
+type Props = {
+  mocks?: MockedResponse[];
+  children?: React.ReactNode;
+};
+
+const TestChild: FC = () => {
+  const { status, data } = useOrganizationListContext();
+
+  if (status === OrgStatus.LOADING) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div>
+      <div data-testid="status">{status}</div>
+      <ul data-testid="organization-list">
+        {data?.map((org, index) => (
+          <li key={org?._id} data-testid={`organization-${index}`}>
```

Comment:
Since `_id` isn't actually provided in the mock data, React is complaining about missing the `key` prop here.

![Screenshot 2024-06-06 at 12 11 25 PM](https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/b8fc1859-514d-444a-83fc-d0f9e280dbc3)

Reply:
Ah, I should have checked these errors. Thanks for catching these!